Sources/SQLiteNIO/SQLiteConnection.swift

```diff
@@ -192,6 +192,34 @@ public final class SQLiteConnection: SQLiteDatabase {
         return promise.futureResult
     }
 
+	public func install(customFunction: SQLiteCustomFunction) -> EventLoopFuture<Void> {
+		logger.debug("Adding custom function \(customFunction.name)")
```

Comment:
These should probably be trace logs
```suggestion
		logger.trace("Adding custom function \(customFunction.name)")
```